docs/customer_management/cp_configuration.md

```diff
@@ -89,3 +89,8 @@ ibexa:
                         match:
                             Identifier\ContentType: confirmation_page
 ```
+
+# Order management
+
+Reviewing pending and past orders in Customer Portal requires that you configure all currencies that any of the customers may use under the `ibexa.system.default.product_catalog.currencies` key. 
```

Comment:
Feel free to rephrase it if needed, we need to emphasize that first configured currency will be the one that the list is showing:
```suggestion
Reviewing pending and past orders in Customer Portal requires that you configure all currencies that any of the customers may use under the `ibexa.system.default.product_catalog.currencies` key. The first currency from the list will be taken for filtering orders list as well as for calculating "Average order" and "Total amount".
```